worker/snapshot.go

```diff
@@ -164,6 +164,12 @@ func (w *grpcWorker) StreamSnapshot(stream pb.Worker_StreamSnapshotServer) error
 	if n == nil {
 		return conn.ErrNoNode
 	}
+
+	raftNode := n.Raft()
+	if raftNode == nil {
```

Comment:
Could you explain here, why it"ll become nil. 
I"m just trying to understand.

Reply:
It doesn't become nil but the node might not be properly initialized so n.Raft() returns nil. I think we've seen this happen after nodes are manually removed via the zero HTTP endpoint.